pkg/diagnostics/network/objects.go

```diff
@@ -131,6 +123,14 @@ func GetTestPod(testPodImage, testPodProtocol, podName, nodeName string, testPod
 	return pod
 }
 
+func getTrimmedImage(image string) string {
+	// Image format could be: [<dns-name>/]openshift/origin-deployer[:<tag>]
+	tokens := strings.Split(image, "/")
```

Comment:
Can't you use https://golang.org/pkg/strings/#SplitN

Reply:
We don't know how many `/`s will be present in the given image (dns path may have many) and we only care about the last token.

Reply:
Oh, the last.  Gotcha!